k8s_audit_config/README.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+This page describes how to get K8s Audit Logging working with the Sysdig Agent. For now, we'll describe how to enable audit logging in k8s 1.11, where the audit configuration needs to be directly provided to the api server. In 1.13 there is a different mechanism that allows audit confguration to be managed like other k8s objects, but these instructions are for 1.11.
+
+The main steps are:
+
+1. Determine if your K8s variant supports audit logging
+1. Deploy the Sysdig Agent to your K8s cluster
+1. Define your audit policy and webhook configuration
+1. Restart the API Server to enable Audit Logging
+1. Observe K8s audit events at the Sysdig Agent
+
+## Determine if your K8s variant supports audit logging
+
+K8s Audit Logging is a relatively new feature, and in k8s 1.11 requires modifying the apiserver configuration. As a result, not every K8s Variant exposes the necessary configuration options to enable and configure audit logging. These instructions have been verified to work with the following K8s Variants:
+
+* Minikube (>= 0.33.1) with the default virtualbox driver.
+* Kops (>= 1.11.0) using AWS.
+
+## Deploy Sysdig Agent to your K8s cluster
+
+Follow the [Agent Installation Instructions for Kubernetes](https://sysdigdocs.atlassian.net/wiki/x/uQB3Cw) to create a Sysdig Agent service account, service, configmap, and daemonset.
+
+## Define your audit policy and webhook configuration
+
+The files in this directory can be used to configure k8s audit logging. The relevant files are:
+
+* [audit-policy.yaml](./audit-policy.yaml): The k8s audit log configuration we recommend that is compatible with the default k8s audit rules published by Sysdig.
+* [webhook-config.yaml.in](./webhook-config.yaml.in): A (templated) webhook configuration that sends audit events to an ip associated with the Sysdig Agent service, port 8765. It is templated in that the *actual* ip is defined in an environment variable `AGENT_SERVICE_CLUSTERIP`, which can be plugged in using a program like `envsubst`.
+
+Run the following to fill in the template file with the ClusterIP ip address you created with the `sysdig-agent` service above. Although services like `sysdig-agent.default.svc.cluster.local` can not be resolved from the kube-apiserver container within the minikube vm (they're run as pods but not *really* a part of the cluster), the ClusterIPs associated with those services are routable.
+
+```
+AGENT_SERVICE_CLUSTERIP=$(kubectl get service sysdig-agent -o=jsonpath={.spec.clusterIP}) envsubst < webhook-config.yaml.in > webhook-config.yaml
+```
+
+## Restart the API Server to enable Audit Logging
```

Comment:
I've noticed it is mentioned in k8s_audit_config/enable-k8s-audit.sh, but nevertheless. For clearness sake, we might want to state that any change to the kube-apiserver.yaml would automatically trigger the API Server restart.

Reply:
Good point, I'll add.